Assets/ScriptableObjects/Plugin Scripts/FragilePlugin.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu]
+public class FragilePlugin : BlockPlugin
+{
+
+    public float TimeToBreak;
+
+    private bool _isDisplaced;
+    private bool _isFirstSteppedOn;
+
+    private void OnEnable()
+    {
+        _isDisplaced = false;
+        _isFirstSteppedOn = false;
+    }
+
+    public override void OnUpdate()
+    {
+        if (!_isFirstSteppedOn && _block.Top.GetCollider().name == "Character")
```

Comment:
This was done using the old blocks which use 6 quads right? @nicholaschuayunzhi does this break with your new block code?

Reply:
It doesn't apply anymore